src/test/scala/uk/gov/hmrc/ct/computations/CP997Spec.scala

```diff
@@ -27,23 +27,23 @@ class CP997Spec extends WordSpec with Matchers with MockitoSugar with BoxValidat
 
   override val boxRetriever = makeBoxRetriever()
 
-  testMandatoryWhen("CP997", CP997.apply) {
-    makeBoxRetriever(cp281bValue = Some(1))
-  }
-
-  testBoxIsZeroOrPositive("CP997", CP997.apply)
-
-  "CP997" should {
-    "fail validation if it exceeds non trading profit" in {
-      CP997(2).validate(makeBoxRetriever(cato01Value = 1)).contains(CtValidation(Some("CP997"), "error.CP997.exceeds.nonTradingProfit")) shouldBe true
-    }
-    "pass validation if it equals non trading profit" in {
-      CP997(2).validate(makeBoxRetriever(cato01Value = 2)).contains(CtValidation(Some("CP997"), "error.CP997.exceeds.nonTradingProfit")) shouldBe false
-    }
-    "pass validation if it is less than non trading profit" in {
-      CP997(1).validate(makeBoxRetriever(cato01Value = 2)).contains(CtValidation(Some("CP997"), "error.CP997.exceeds.nonTradingProfit")) shouldBe false
-    }
-  }
+//  testMandatoryWhen("CP997", CP997.apply) {
+//    makeBoxRetriever(cp281bValue = Some(1))
+//  }
+//
+//  testBoxIsZeroOrPositive("CP997", CP997.apply)
+//
+//  "CP997" should {
+//    "fail validation if it exceeds non trading profit" in {
+//      CP997(2).validate(makeBoxRetriever(cato01Value = 1)).contains(CtValidation(Some("CP997"), "error.CP997.exceeds.nonTradingProfit")) shouldBe true
+//    }
+//    "pass validation if it equals non trading profit" in {
+//      CP997(2).validate(makeBoxRetriever(cato01Value = 2)).contains(CtValidation(Some("CP997"), "error.CP997.exceeds.nonTradingProfit")) shouldBe false
+//    }
+//    "pass validation if it is less than non trading profit" in {
+//      CP997(1).validate(makeBoxRetriever(cato01Value = 2)).contains(CtValidation(Some("CP997"), "error.CP997.exceeds.nonTradingProfit")) shouldBe false
+//    }
+//  }
```

Comment:
same here

Reply:
Removed it because they removed the validation from the box CP997 and changed to be the pre-populated field.